Examples/Sources/MacroExamples/Implementation/Peer/AddAsyncMacro.swift

```diff
@@ -42,7 +42,9 @@ public struct AddAsyncMacro: PeerMacro {
     }
 
     // This only makes sense void functions
-    if funcDecl.signature.returnClause?.type.with(\.leadingTrivia, []).with(\.trailingTrivia, []).description != "Void" {
+    guard let returnIdentifier = funcDecl.signature.returnClause?.type.as(IdentifierTypeSyntax.self),
+      returnIdentifier.name.text == "Void"
+    else {
```

Comment:
Much nicer. Is there any particular reason why you chose to store the `returnIdentifier` instead of doing

```suggestion
    guard funcDecl.signature.returnClause?.type.as(IdentifierTypeSyntax.self).name.text == "Void" else {
```

Reply:
Oops, my brain took a coffee break there! No real reason, just one of those moments. 😄